src/main/java/com/github/teamfusion/spyglassplus/client/model/SmallSpyglassStandModel.java

```diff
@@ -0,0 +1,56 @@
+package com.github.teamfusion.spyglassplus.client.model;// Made with Blockbench 4.2.5
+// Exported for Minecraft version 1.17 - 1.18 with Mojang mappings
+// Paste this class into your mod and generate all required imports
+
+import com.github.teamfusion.spyglassplus.common.entity.SpyglassStandEntity;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.blaze3d.vertex.VertexConsumer;
+import net.minecraft.client.model.EntityModel;
+import net.minecraft.client.model.geom.ModelPart;
+import net.minecraft.client.model.geom.PartPose;
+import net.minecraft.client.model.geom.builders.*;
```

Comment:
A wildcard import shouldn't be necessary here. I'd replace this with the separate necessary imports